app/helpers/application_helper.rb

```diff
@@ -216,7 +216,7 @@ def visits_select_options(arm, pages)
       (beginning_visit..ending_visit).each do |visit_number|
         visit_group = arm.visit_groups[visit_number - 1]
 
-        if visit_group.day.present?
+        unless visit_group.day.nil?
```

Comment:
I'm fine either way. I think I prefer it the way it is. Both ways make sense to me.